docs/config.json

```diff
@@ -567,17 +567,22 @@
           "slug": "/management/export-audit-events/",
           "entries": [
             {
-              "title": "Exporting Audit Events to Fluentd",
+              "title": "Export Audit Events to Fluentd",
               "slug": "/management/export-audit-events/fluentd/",
               "forScopes": ["enterprise", "cloud"]
             },
             {
-              "title": "Monitoring Audit Events with the Elastic Stack",
+              "title": "Export Audit Events with Datadog",
```

Comment:
Export was chosen by @zmb3 in his edit.

Reply:
Would "Export ... to Datadog" work for you?

Reply:
I love it! The only way it could be better is if it was already in a suggestion, so I could have just clicked a button to make it happen.

```suggestion
              "title": "Export Audit Events to Datadog",
```